### ISSUE TITLE
How to resume the training

### ISSUE THREAD

**Issue by ax642511** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#262]
Hi，teacher，Excuse me, I have used mobilenet weight to train the network for 20000 iter。And,i want to start from 20000 iter continuing trainning. I use "python train.py --train-images-folder ./coco/train2017/ --prepared-train-labels prepared_train_annotation.pkl --val-labels val_subset.json --val-images-folder ./coco/val2017/ --checkpoint-path ./_checkpoints/checkpoint_iter_20000.pth --weights-only".But theprogram still starts training from 100iter。thank you .
![1](https://user-images.githubusercontent.com/73568911/166178133-31c004ad-a17c-472d-bef9-164f58929a71.png)

Another question, i study for longtime ,i don't understand the difference between step 4 and step 5,what is the "--weights-only --num-refinement-stages 3"

**Comment by Daniil-Osokin** [comment]
Hi! Training will continue automatically when the pre-trained checkpoint is provided. Just remove `--weights-only` flag. Step 4 shows how to start training from the already trained model, `--weights-only` flag allows to copy network weights while forget training statistics. This gives better initial weights estimation in comparison with bare MobileNet weights. Step 5 shows how to add additional refinement stages, which makes network deeper. Both steps improve network accuracy.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.

**Comment by sulei1998** [comment]
When i want to resume the training. I met this problem.
![image](https://user-images.githubusercontent.com/56378342/205587185-b484ca54-b567-4dcf-a994-26b00b58a6f1.png)
